src/IntegratedExperts/BehatScreenshot/ScreenshotExtension.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/**
+ * @file
+ * Behat screenshot extension.
+ */
+
+namespace IntegratedExperts\BehatScreenshot;
+
+use Behat\Behat\Context\ServiceContainer\ContextExtension;
+use Behat\Testwork\ServiceContainer\Extension as ExtensionInterface;
+use Behat\Testwork\ServiceContainer\ExtensionManager;
+use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+
+/**
+ * Class ScreenshotExtension
+ */
+class ScreenshotExtension implements ExtensionInterface
+{
+    /**
+     * Extension configuration ID.
+     */
+    const MOD_ID = 'integratedexperts_screenshot';
+
+    /**
+     * Default base path.
+     */
+    const BASE_PATH = '%paths.base%';
+
+    /**
+     * Default screenshot dir.
+     */
+    const DEFAULT_SCREENSHOT_DIR = self::BASE_PATH.'/screenshot';
```

Comment:
we do not want any defaults - this should be set explicitly in config or an exception should be thrown